### PR TITLE
Add broshure example

### PR DESCRIPTION
**Pull request by yurybotov**
Add quick-start-node **segm8-number**, **example-counter** for broshure. And rename DO pulses to UPD to avoid confusion with DO input - SPI Data Out.

### CODE REVIEW

File: SegM8/clear/patch.xodp [hunk 1]
```diff
@@ -4,7 +4,7 @@
     {
       "description": "Triggers a new clear.",
       "id": "BJkYMOrbv",
-      "label": "DO",
```
Comment (evgenykochetkov) [inline SegM8/clear/patch.xodp:7]: The pins were renamed, but the C++ implementation is not updated.
Things like `isInputDirty<input_DO>(ctx)` won't compile.

**Review by evgenykochetkov — CHANGES_REQUESTED** [review]

File: SegM8/clear/patch.xodp (hunk 1 above)
Comment (yurybotov) [reply SegM8/clear/patch.xodp:7]: Thanks! I will fix it. And will add 'any' to q-s-n

Comment (yurybotov) [reply SegM8/clear/patch.xodp:7]: Done

**Review by evgenykochetkov — APPROVED** [review]
LGTM!

**Review by nomad605dis — APPROVED** [review]
LGTM

File: SegM8/segm8-number/patch.xodp [hunk 2]
```diff
@@ -0,0 +1,222 @@
+{
+  "description": "Simple quick-start-node. Displays a number value at the SegM8 module or several modules linked in a daisy chain.",
```
Comment (nkrkv) [inline SegM8/segm8-number/patch.xodp]:
```suggestion
  "description": "Displays a number value on a SegM8 module or several modules linked in a daisy chain.",
```

File: SegM8/segm8-number/patch.xodp [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        "x": 11,
+        "y": 1
+      },
+      "type": "xod/patch-nodes/input-boolean"
+    },
+    {
+      "boundLiterals": {
+        "__out__": "D8"
+      },
+      "description": "The CS is a signal that controls the display of the value transferred to the SegM8. You can use any Arduino pin. For example: 'D8'.",
```
Comment (nkrkv) [inline SegM8/segm8-number/patch.xodp]:
```suggestion
      "description": "The board port to which the CS pin of SegM8 is connected. You may use any Arduino pin. For example: 'D8'.",
```

File: SegM8/segm8-number/patch.xodp [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+      "label": "DO",
+      "position": {
+        "units": "slots",
+        "x": 7,
+        "y": 1
+      },
+      "type": "xod/patch-nodes/input-port"
+    },
+    {
+      "description": "The number of modules in the daisy-chain. The default is 1.",
```
Comment (nkrkv) [inline SegM8/segm8-number/patch.xodp]:
```suggestion
      "description": "The number of modules in the daisy-chain.",
```

File: SegM8/segm8-number/patch.xodp [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+        "x": 3,
+        "y": 3
+      },
+      "type": "@/segm8-device"
+    },
+    {
+      "boundLiterals": {
+        "__out__": "D255"
+      },
+      "description": "The DO is an Arduino pin through which data is transmitted with software SPI. You can use any pin of the Arduino. For example: 'D3'.\nLeave this parameter at 'D255' when using hardware SPI.",
```
Comment (nkrkv) [inline SegM8/segm8-number/patch.xodp]:
```suggestion
      "description": "The board port to which the DO pin of SegM8 is connected. You may use any Arduino pin. For example: 'D3'.\nLeave this parameter at 'D255' when using hardware SPI.",
```

File: SegM8/segm8-number/patch.xodp [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+        "x": 1,
+        "y": 5
+      },
+      "type": "@/display-number"
+    },
+    {
+      "boundLiterals": {
+        "__out__": "D255"
+      },
+      "description": "The CLK is the Arduino pin through which the clock signals are transmitted with software SPI. You can use any pin of the Arduino. For example: 'D4'.\nLeave this parameter at 'D255' when using hardware SPI.",
```
Comment (nkrkv) [inline SegM8/segm8-number/patch.xodp]:
```suggestion
      "description": "The board port to which the CLK pin of SegM8 is connected. You may use any Arduino pin. For example: 'D4'.\nLeave this parameter at 'D255' when using hardware SPI.",
```

File: SegM8/example-counter/patch.xodp [hunk 7]
```diff
@@ -0,0 +1,66 @@
+{
+  "description": "Example of use quick-start-node segm8-number.",
```
Comment (nkrkv) [inline SegM8/example-counter/patch.xodp]:
```suggestion
  "description": "Quickstart usage example.",
```

**Review by nkrkv — CHANGES_REQUESTED** [review]

File: SegM8/segm8-number/patch.xodp (hunk 2 above)
Comment (yurybotov) [reply SegM8/segm8-number/patch.xodp]: OK

File: SegM8/segm8-number/patch.xodp (hunk 3 above)
Comment (yurybotov) [reply SegM8/segm8-number/patch.xodp]: OK

File: SegM8/segm8-number/patch.xodp (hunk 4 above)
Comment (yurybotov) [reply SegM8/segm8-number/patch.xodp]: OK.

File: SegM8/segm8-number/patch.xodp (hunk 5 above)
Comment (yurybotov) [reply SegM8/segm8-number/patch.xodp]: OK

File: SegM8/segm8-number/patch.xodp (hunk 6 above)
Comment (yurybotov) [reply SegM8/segm8-number/patch.xodp]: OK

File: SegM8/example-counter/patch.xodp (hunk 7 above)
Comment (yurybotov) [reply SegM8/example-counter/patch.xodp]: OK

**Comment by yurybotov** [comment]
All changes accepted

**Review by nkrkv — APPROVED** [review]
:ok_hand:

**Pull request merged**